nighthawk/source/common/rate_limiter_impl.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "common/common/logger.h"
+#include "envoy/event/timer.h"
+
+#include "nighthawk/common/rate_limiter.h"
```

Comment:
Re: header order, would be great to get a convention and have a check_format/fix_format script to enforce like on Envoy side.